KSP-Backend/backend/api/models.py

```diff
@@ -0,0 +1,53 @@
+from django.db import models
+from vote.models import VoteModel
+import datetime
+#from vote.models import VoteModel
+
+# Create your models here.
+YEAR_CHOICES = []
+for year in range(2016, (datetime.datetime.now().year + 1)):
+    YEAR_CHOICES.append((year, year))
+
+
+
+class PreviousYearQuestions(models.Model):
+    title = models.CharField(max_length=120, default="")
+    name = models.CharField(max_length=100, default="")
+    course_id = models.CharField(max_length=10, default="")
+    batch = models.IntegerField(choices=YEAR_CHOICES, default="")
+    course_instructor = models.CharField(max_length=100, default="")
+    #uploaded_at = models.DateTimeField(auto_now=True)
+    gdrive_link = models.URLField(max_length=500, default="")
+    completed = models.BooleanField(default=False)
+
+    def _str_(self):
+        return self.title
+
+class Notes(VoteModel, models.Model):

```

Comment:
@Ananyaiitbhilai As I previously commented, having two models with exact same schema is not a good practice. If you want a distinction between the two, just add another field(say, a `category` or `notes_type` field that can either if two values, notes or previous year questions). Later when you need to filter between these two, you can use this field category for that. Fix this, please.